diff --git a/node_modules/@react-native-community/async-storage/RNCAsyncStorage.podspec b/node_modules/@react-native-community/async-storage/RNCAsyncStorage.podspec
index ec75873..f8d6504 100644
--- a/node_modules/@react-native-community/async-storage/RNCAsyncStorage.podspec
+++ b/node_modules/@react-native-community/async-storage/RNCAsyncStorage.podspec
@@ -16,4 +16,5 @@ Pod::Spec.new do |s|
   s.source_files  = "ios/**/*.{h,m}"
 
   s.dependency 'React'
+  s.dependency 'React-Core'
 end
